smsdk/ma_session.py

```diff
@@ -96,8 +127,7 @@ def _get_records(
                 
             except:
                 import traceback
-                
-                print(traceback.print_exc())
+                log.error(traceback.print_exc())
```

Comment:
we should mark this as warning If it's returning records

Reply:
Added warning